### ISSUE TITLE
Unknown or unsupported action after closing and opening the widget

### ISSUE THREAD

**Issue by p1gp1g** [vector-im/element-web#21174]
### Steps to reproduce

0. When using [https://github.com/maunium/stickerpicker](stickerpicker) as a widget
`/addwidget https://maunium.net/stickers-demo/`
1. Clicking on a sticker sends it
2. Closing the widget and opening again
3. Clicking on a sticker throws the following error:
```
{
  "api": "fromWidget",
  "action": "m.sticker",
  "requestId": "sticker-1645619199735",
  "widgetId": "customwidget_<redacted>",
  "data": {
    "content": {
      "body": "🤫",
      "info": {
        "h": 300,
        "mimetype": "image/png",
        "size": 258227,
        "w": 563,
        "xyz.amorgan.blurhash": "LJFEJ@RQnh9c|2nhn$aLGvxCn$-n"
      },
      "msgtype": "m.image",
      "url": "mxc://matrix.org/<redacted>"
    },
    "name": "🤫"
  },
  "widgetData": {
    "content": {
      "body": "🤫",
      "info": {
        "h": 300,
        "mimetype": "image/png",
        "size": 258227,
        "w": 563,
        "xyz.amorgan.blurhash": "LJFEJ@RQnh9c|2nhn$aLGvxCn$-n"
      },
      "msgtype": "m.image",
      "url": "mxc://matrix.org/<redacted>"
    },
    "name": "🤫",
    "description": "🤫",
    "file": "<redacted>"
  },
  "response": {
    "error": {
      "message": "Unknown or unsupported action: m.sticker"
    }
  }
}
```

#### What did you expect?
The sticker is sent

#### What happened instead?
An error occured

### Browser information

Mozilla Firefox 97.0.1, Desktop app

### URL for webapp

app.element.io

### Application version

Element : 1.10.4, Olm : 3.2.8

### Homeserver

matrix.org

### Will you send logs?

If needed

**Comment by gsouquet** [comment]
Could you get a copy of the payload sent to the widget before and a copy of the payload sent after?
That might highlight a difference of attributes sent to the widget

**Comment by p1gp1g** [comment]
* Opening
```
{
  "api": "toWidget",
  "widgetId": "!aijGWHQNrZdqSWaDqW%3Amatrix.org_%40s%3Amatrix.org_1645634306166",
  "requestId": "widgetapi-1645634356896",
  "action": "capabilities",
  "data": {}
}
```
```
{
  "api": "toWidget",
  "widgetId": "!aijGWHQNrZdqSWaDqW%3Amatrix.org_%40s%3Amatrix.org_1645634306166",
  "requestId": "widgetapi-1645634356921",
  "action": "notify_capabilities",
  "data": {
    "requested": [
      "m.sticker"
    ],
    "approved": [
      "m.sticker"
    ]
  }
}
```
* Clicking on sticker
```
{
  "api": "fromWidget",
  "action": "m.sticker",
  "requestId": "sticker-1645634369884",
  "widgetId": "!aijGWHQNrZdqSWaDqW%3Amatrix.org_%40s%3Amatrix.org_1645634306166",
  "data": {
    "content": {
      "body": "🎓",
      "url": "mxc://maunium.net/pxbCqPJNvcFIZlAyKCyXsqfJ",
      "info": {
        "w": 256,
        "h": 256,
        "size": 106237,
        "mimetype": "image/png",
        "thumbnail_url": "mxc://maunium.net/pxbCqPJNvcFIZlAyKCyXsqfJ",
        "thumbnail_info": {
          "w": 256,
          "h": 256,
          "size": 106237,
          "mimetype": "image/png"
        }
      }
    },
    "name": "🎓"
  },
  "widgetData": {
    "content": {
      "body": "🎓",
      "url": "mxc://maunium.net/pxbCqPJNvcFIZlAyKCyXsqfJ",
      "info": {
        "w": 256,
        "h": 256,
        "size": 106237,
        "mimetype": "image/png",
        "thumbnail_url": "mxc://maunium.net/pxbCqPJNvcFIZlAyKCyXsqfJ",
        "thumbnail_info": {
          "w": 256,
          "h": 256,
          "size": 106237,
          "mimetype": "image/png"
        }
      }
    },
    "name": "🎓",
    "description": "🎓",
    "file": "tg-551004416715522189.png"
  },
  "response": {}
}
```
* Closing : nothing
* Opening : nothing
* Clicking on sticker:
```
{
  "api": "fromWidget",
  "action": "m.sticker",
  "requestId": "sticker-1645634567019",
  "widgetId": "!aijGWHQNrZdqSWaDqW%3Amatrix.org_%40s%3Amatrix.org_1645634306166",
  "data": {
    "content": {
      "body": "🎓",
      "url": "mxc://maunium.net/pxbCqPJNvcFIZlAyKCyXsqfJ",
      "info": {
        "w": 256,
        "h": 256,
        "size": 106237,
        "mimetype": "image/png",
        "thumbnail_url": "mxc://maunium.net/pxbCqPJNvcFIZlAyKCyXsqfJ",
        "thumbnail_info": {
          "w": 256,
          "h": 256,
          "size": 106237,
          "mimetype": "image/png"
        }
      }
    },
    "name": "🎓"
  },
  "widgetData": {
    "content": {
      "body": "🎓",
      "url": "mxc://maunium.net/pxbCqPJNvcFIZlAyKCyXsqfJ",
      "info": {
        "w": 256,
        "h": 256,
        "size": 106237,
        "mimetype": "image/png",
        "thumbnail_url": "mxc://maunium.net/pxbCqPJNvcFIZlAyKCyXsqfJ",
        "thumbnail_info": {
          "w": 256,
          "h": 256,
          "size": 106237,
          "mimetype": "image/png"
        }
      }
    },
    "name": "🎓",
    "description": "🎓",
    "file": "tg-551004416715522189.png"
  },
  "response": {
    "error": {
      "message": "Unknown or unsupported action: m.sticker"
    }
  }
}
```

I hope it was what you needed. There isn't much more information from the logs

**Comment by p1gp1g** [comment]
Fixed with Element 1.10.5.
